Packs/AdminByRequest/README.md

```diff
@@ -1,2 +1,76 @@
+# Admin By Request
+
 AdminByRequest is a Privileged Access Management (PAM) solution that enables secure, temporary elevation to local admin rights.
-This integration collects events from the AdminByRequest API.
+
+<~XSIAM>
+
+## What does this pack contain?
+
+- API Integration.
+- Modeling Rules for the following events:
+  - Audit Log
+  - Request
+  - Events
+
+## Configuration on Server Side
+
+### Admin By Request configuration
+
+1. Log into your Admin By Request portal.
+2. Navigate to **Settings** -> **Tenant Settings** -> **API Keys** -> **API KEYS**.
+3. Click **Add New**
+4. Copy and store the generated key,
+
+For more inofrmation use the following guide [here](https://docs.adminbyrequest.com/integrations/public-api.htm).
+
+### Configuration on XSIAM
+
+1. Navigate to **Settings** > **Configuration** > **Data Collection** > **Automation & Feed Integrations**.
+2. Search for "Admin By Request" and click **Add Instance**
+3. When configuring the API Integration, set the following values:
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Server URL |  | True |
+| API Key | The API Key allows you to programmatically integrate with the Armis ecosystem. | True |
+| Trust any certificate (not secure) |  | False |
+| Use system proxy settings |  | False |
+| Fetch events |  | False |
+| Event types to fetch | Which records the integration should fetch from the AdminByRequest API. Available for Auditlogs, Events, and Requests. | True |
```

Comment:
Returns the types of events from the AdminByRequest API. Available for Auditlogs, Events, and Requests. | True |